### DISCUSSION TITLE
Show attempts on a given beatmap

### DISCUSSION THREAD

**Discussion by bytenekos** [ppy/osu#29569] [Ideas] [answered]
Hi there!
Could be cool if attempts were shown on a particular beatmap while in song select somewhere

Could be hidden behind a hover tooltip or shown in the details tab maybe, idk

Idea came to mind when I passed a hard map that I've never passed before so I got curious how many times have I actually attempted to pass that map

**Comment by Tenexxt** [comment]
Technically there is a attempts number when you restart the map, but it resets when you leave to the song select.
A total attempts number is technically on your profile, but looking for a certain map can be annoying so adding this seems good to me

**Reply by bytenekos** [reply]
> A total attempts number is technically on your profile, but looking for a certain map can be annoying

Yup, that's basically the whole reasoning behind this idea ^^

**Comment by Stoppedpuma** [comment, marked as answer]
#28926
